app/Models/Litter/Categories/Brand.php

```diff
@@ -198,18 +79,15 @@ public function types ()
             'supermacs',
             'supervalu',
             'starbucks',
-
             'tayto',
             'tesco',
             'thins',
-
             'volvic',
-
             'waitrose',
             'walkers',
             'woolworths',
             'wilde_and_greene',
-            'wrigleys'
+            'wrigleys',
```

Comment:
please remove the trailing commas

Reply:
I generally like to keep these trailing commas in arrays like these. I've seen it in other open-source repositories too. They do it so that when you add a new item, the git diffs show only a new line added, not two lines changed. I can remove the comma, but I think it's better like this. Should I remove them?

Reply:
I don't mind. 1 new line in git diff makes sense